main.go

```diff
@@ -233,6 +236,8 @@ Examples:
 	--filter c                                                    | only trace events from containers (same as above)
 	--filter !container                                           | only trace events from the host
 	--filter '!c'                                                 | only trace events from the host (same as above)
+	--filter close:fd=5                                           | only trace 'close' events that have 'fd' equals 5
+	--filter openat:pathname!=/tmp/1,/bin/ls                      | don't trace 'openat' events that have 'pathname' equals /tmp/1 or /bin/ls
```

Comment:
I see that you allow multiple values in the same filter.
1. this is not mentioned in the help (outside the example)
2. it should be clear what `,` means - AND/OR. perhaps replace it with a symbol that better reflects
3. I think it would be better to not support multiple values, just like we don't allow for other filters. if someone wants to filter multiple values, they can specify the filter multiple times (like with other filters)

Reply:
Going backward:
3. All of our filters support multiple values: uid, pid, comm, uts... Why do you say they don't?
2. That depends. It leads me back to the discussion with @grantseltzer about the expected filter behavior. If '=' is chosen, it means logical OR, if '!=' is chosen it means logical AND. This is also why ',' suits here best
1. The example is the help. I can add a comment if you think that is necessary

Reply:
I do agree that ',' is cleaner looking but I can't think of other tools that have a similar interface where they wouldn't have to read the docs anyway.

Reply:
if you're keeping `,`. can you clarify the behavior in the help?

Reply:
> The example is the help. I can add a comment if you think that is necessary

The example should demonstrate what's defined in the help. I do think it should be clearly defined when discussing the filter